### DISCUSSION TITLE
Change git command in README

### DISCUSSION THREAD

**Discussion by epogrebnyak** [gethyas/hyas#42] [Ideas]
Maybe change `git clone git@github.com:h-enk/hyas.git my-hyas-site` to more friendly `git clone https://github.com/h-enk/hyas.git my-hyas-site`.

 `git clone git@github.com:h-enk/hyas.git my-hyas-site` gieve me access errors, while `git clone https://github.com/h-enk/hyas.git my-hyas-site` works fine.

**Comment by h-enk** [comment]
Thanks, I will add it the HTTPS clone as the first option — next to the SSH (and GitHub CLI) option.

**Comment by h-enk** [comment]
To keep things simple, HTTPS is now the only clone option recorded in the README and documentation. Those (advanced users) who want to use the SSH or GitHub CLI option will know how how — is my assumption.
